pkgs/applications/emulators/box64/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "box64";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "ptitSeb";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-6k8Enbafnj19ATtgmw8W7LxtRpM3Ousj1bpZbbtq8TM=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    python3
+  ];
+
+  cmakeFlags = [
+    "-DNOGIT=1"
+    "-DCMAKE_BUILD_TYPE=Release"
```

`Release` is the default so it could be removed. If you'd prefer to leave it in case it's ever flipped to something else, please use [`cmakeBuildType`](https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L107):

```suggestion
  cmakeBuildType = "Release";
  
  cmakeFlags = [
    "-DNOGIT=1"
```

---

actually it seems it isn't: https://github.com/ptitSeb/box64/blob/3dec81b9ebe856f3575f7aa68f8a099d8a99fc39/CMakeLists.txt#L78

---

or will our setuphook override the default of the package?